server/main.go

```diff
@@ -70,16 +83,27 @@ func serve(port int, errc chan error) error {
 					w.WriteHeader(http.StatusNoContent)
 					return
 				}
+				if req.URL.Path == "/"+client {
+					clientfile, err := os.Open(client)
```

Comment:
let's use `http.ServeFile` here which will handle for us all the edge cases (proper headers, handling errors, etc.)